integration-test/test.bats

```diff
@@ -71,21 +71,50 @@
 
 @test "govvv build - program with compile-time variables" {
     tmp="${BATS_TMPDIR}/a.out"
-    run govvv build -o "$tmp" ./integration-test/app-example  
+    run bash -c "cd ${BATS_TEST_DIRNAME}/app-example && govvv build -o $tmp"
```

Comment:
why do you need to change this line? it was working fine.

Reply:
Correct, don't need it.  More WIP leakage.  :(